src/sentry/models/releaseprojectenvironment.py

```diff
@@ -19,9 +19,18 @@ class ReleaseProjectEnvironment(Model):
     last_seen = models.DateTimeField(default=timezone.now, db_index=True)
     last_deploy_id = BoundedPositiveIntegerField(null=True, db_index=True)
 
+    adopted = models.DateTimeField(null=True, blank=True)
+    unadopted = models.DateTimeField(null=True, blank=True)
+
     class Meta:
         app_label = "sentry"
         db_table = "sentry_releaseprojectenvironment"
+        index_together = (
+            ("project", "adopted"),
+            ("project", "adopted", "environment"),
+            ("project", "unadopted"),
```

Comment:
You can just keep the indexes that have `environment` and remove the others, queries that only use `project, (un)adopted` can make use of the environment indexes as well